.pre-commit-config.yaml

```diff
@@ -18,3 +18,27 @@ repos:
     rev: 6.1.0
     hooks:
     - id: flake8
+  - repo: https://github.com/astral-sh/ruff-pre-commit

```

Comment:
Good question, I think it could replace the package above, but I need to double check if that depend on certain settings. I think for now I just replaces flake8, but I will look into it and remove the replaced packages.

Reply:
Yeah, checked and replaced all the packages. The import sort is slightly different but it only made a difference in 3 files, so to me it didn't seem like an issue and worth it to only use ruff.